framework/templates/ecl-filters/ecl-filters.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  title: 'Filters',
+  label: 'Filters',
+  status: 'ready',
+  tags: ['template'],
+  context: {
+    formID: 'form-filter',
```

Comment:
I don't see a reason to have this is via the context having the fact that the rest is directly inside the template. And since the rest are inside, it will be easier to see the relation even.